### DISCUSSION TITLE
auto map [Feb 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5061] [Discord Threads]
> 🙋‍♂️ **Deleted User**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### Can you provide more details about your requirements?  
#  
> 🙋‍♂️ **Kannan**  
####   
![image0](https://cdn.discordapp.com/attachments/1059348565635366923/1059354524097192006/image.png)  
#  
> 🙋‍♂️ **Kannan**  
####   
![image0](https://cdn.discordapp.com/attachments/1059348565635366923/1059354611615535377/image.png)  
#  
> 🙋‍♂️ **Kannan**  
#### This is the school id that i assigned in here also i set it as primary key , same like it is in another may more tables , i want to make it assign automatically , is that it possible , is ther anyway to do it .  
#  
> 🙋‍♂️ **Kannan**  
####   
![image0](https://cdn.discordapp.com/attachments/1059348565635366923/1059355254933700648/image.png)  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### If you are creating links ("LinkToAnotherRecord"), you can use look up to refer to linked table fields (LookUp: https://docs.nocodb.com/setup-and-usages/lookup)   
####    
#### Auto populating a field based on linked record- is not possible. We do have formula's- but you can reference only fields within the table when creating a formula.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1059348565635366923)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
